app/src/components/Generation/FloatingGenerateBox.tsx

```
@@ -16,7 +16,7 @@ import {
 import { Textarea } from '@/components/ui/textarea';
 import { useToast } from '@/components/ui/use-toast';
 import { apiClient } from '@/lib/api/client';
-import { getLanguageOptionsForEngine, type LanguageCode } from '@/lib/constants/languages';
+import { LANGUAGE_OPTIONS, getLanguageOptionsForEngine, type LanguageCode } from '@/lib/constants/languages';
```

Biome (2.4.14)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)